src/ports/bids/queries.ts

```diff
@@ -62,10 +62,31 @@ export function getBidTradesQuery() {
 
 export function getBidsQuery(options: GetBidsParameters) {
   const FROM_BID_TRADES = SQL` FROM (`.append(getBidTradesQuery()).append(SQL`) as bid_trades`)
-  const WHERE_BIDDER = options.bidder ? SQL` WHERE bidder = ${options.bidder}` : SQL``
+
+  const FILTER_BY_BIDDER = options.bidder ? SQL` LOWER(bidder) = LOWER(${options.bidder})` : null
```

Comment:
Is it missing a whitespace here?
```suggestion
  const FILTER_BY_BIDDER = options.bidder ? SQL` LOWER(bidder) = LOWER(${options.bidder}) ` : null
```